src/views/guidelines/guidelines.jsx

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+var render = require('../../lib/render.jsx');
+var Page = require('../../components/page/page.jsx');
+
+require('./guidelines.scss');
+
+var Guidelines = React.createClass({
+    type: 'Guidelines',
+    render: function () {
+        return (
+            <div className="body">
+                <div className="top">
+                    <h1>Scratch Community Guidelines</h1>
+                </div>
+                <div className="bottom">
+                    <p>We need everyone's help to keep Scratch a friendly and creative
```

Comment:
The translation part of this needs the `FormattedMessage` component, rather than static text like this. See the [localization guide](https://github.com/LLK/scratch-www/wiki/Localization-Guide) for more info on what render looks like with localized strings, and let me know if you have any questions!